.github/workflows/benchmark.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: ['10.x', '12.x', '14.x']
+        node-version: ['10', '12', '14']
```

Comment:
Surge.sh's subdomain cannot have a dot, `10` & `10.x` seem to produce similar result.